packages/api/src/db/peer.ts

```diff
@@ -3,7 +3,8 @@ import {RichTextNode} from '../graphql/richText'
 export interface PeerProfile {
   name: string
   logoID?: string
-  themeColor: string
+  themeColor?: string
```

Comment:
Do you want it optional here?

Reply:
I suppose it's more consistant if it's not optional, I changed it